eth/protocols/eth/protocol.go

```diff
@@ -212,6 +235,12 @@ type BlockBodiesPacket66 struct {
 	BlockBodiesPacket
 }
 
+// BlockBodiesPacket100 is the network packet for block content distribution over eth/100.
+type BlockBodiesPacket100 struct {
+	RequestId   uint64
+	BlockBodies []BlockBody100
```

Comment:
I think we should use []*BlockBody100, which is potentially faster when passing/copying this struct around.